recipes/libsystemd/all/conandata.yml

```diff
@@ -14,15 +14,18 @@ sources:
   "251.4":
     url: "https://github.com/systemd/systemd-stable/archive/v251.4.tar.gz"
     sha256: "3459239979f52b8c4ace33734d31c2fb69fa13cf81d04b1b982f7d8d4651e015"
+  "252.4":
+    url: "https://github.com/systemd/systemd-stable/archive/v252.4.tar.gz"
+    sha256: "cf2d27e67663d599a045101c7178cf0ec63d9df2962a54adf7de0d0357724f00"
 patches:
   "246.16":
     - patch_file: "patches/246.16/0001-Drop-bundled-copy-of-linux-if_arp.h.patch"
       patch_description: "fix build error with Linux headers >= 5.14 by removing a bundled copy of it"
-      patch_type: "backport"
+      patch_type: "bugfix"
```

Comment:
I think it's the portability patch type instead of bugfix.

Reply:
Probably it is.
Honestly I don't really like both versions, but I don't remember why I decided to use bugfix here.

And just for context:
```
patch_type: portability: Indicates a patch that improves the portability of the library, e.g. adding supports of new architectures (ARM, Sparc, etc.), operating systems (FreeBSD, Android, etc.), compilers (Intel, MinGW, etc.), and other types of configurations which are not originally supported by the project. ... Some projects simply do not accept patches for platforms they don't have a build/test infrastructure, or maybe they are just either abandoned or inactive.
```
```
patch_type: bugfix: Indicates a patch that backports an existing bug fix from the newer release or master branch (or equivalent, such as main/develop/trunk/etc).
```